mteb/tasks/Retrieval/fas/NeuCLIR2022FasRetrieval.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class NeuCLIR2022FasRetrieval(AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="NeuCLIRFas2022",
+        description="The task involves identifying and retrieving the documents that are relevant to the queries.",
+        reference="https://neuclir.github.io/",
+        dataset={
+            "path": "mteb/neuclir-2022-fas",
+            "revision": "fa3b9068b9318d13ed5f065d9bbd5c8aff2dcfec",
+        },
+        type="Retrieval",
+        category="s2p",
+        eval_splits=["test"],
+        eval_langs=["fas-Arab"],
```

Comment:
Assuming you combine these datasets you will have to specify eval_langs as a dict:
```suggestion
        eval_langs={"fas": "fas-Arab", 
                              "rus": ...},
```
Where rus and fas are the language keys used for the dataset. You can also do a list if you use the exact same keys.